api/src/tests/simulation_test.rs

```diff
@@ -82,3 +86,57 @@ async fn test_simulate_transaction_with_insufficient_balance() {
     let resp = simulate_aptos_transfer(&mut context, false, LARGE_TRANSFER_AMOUNT, 200).await;
     assert!(!resp[0]["success"].as_bool().is_some_and(|v| v));
 }
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
Do we know what is this `worker_threads = 2` achieving?

Reply:
This makes sure your runtime is configured as multi-threaded with 2 threads to run async